build-logic/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle

```diff
@@ -386,13 +393,13 @@ performanceTest.registerTestProject("bigSwiftApp", BuildBuilderGenerator) {
 }
 
 // === Android ===
-performanceTest.registerTestProject("largeAndroidBuild", RemoteProject) {
+performanceTest.registerAndroidTestProject("largeAndroidBuild", RemoteProject) {
     remoteUri = 'https://github.com/gradle/perf-android-large.git'
     // From android-41 branch
     ref = "b3964ffea18ef5d280b5b493c5c7b8d8b986056c"
 }
 
-performanceTest.registerTestProject("santaTrackerAndroidBuild", RemoteProject) {
+performanceTest.registerAndroidTestProject("santaTrackerAndroidBuild", RemoteProject) {
```

Comment:
The `uberMobileApp` is also an Android test project, so itt should use the method as well.

Reply:
Fixed